addon-test-support/@ember/test-helpers/wait-until.js

```diff
@@ -18,9 +18,10 @@ const MAX_TIMEOUT = 10;
 */
 export default function waitUntil(callback, options = {}) {
   let timeout = 'timeout' in options ? options.timeout : 1000;
+  let errorMsg = 'errorMessage' in options ? options.errorMessage : 'waitUntil timed out';
```

Comment:
lets call this explicitly `timeoutMessage` (we may have other types of errors in the future, so I'd like to avoid using the more generic name for now)